### ISSUE TITLE
Avoid prompting for the role if the role is given

### ISSUE THREAD

**Issue by froshyfrosh** [oktadeveloper/okta-aws-cli-assume-role#246]
Hello,
thanks for the nice work. I'm not sure whether this should be a bug report or a feature request. It would be great if something could be included that helps me here. 
Thanks in advance, Patrick

**Describe the bug**
Although the role is given by the OktaAwsCliEnvironment the user is prompted for selecting the role

**To Reproduce**
No profile yet created
Create an instance of OktaAwsCliAssumeRole with a OktaAwsCliEnvironment containing all necessary attributes (incl. the role)
call OktaAwsCliAssumeRole#getAssumeRoleWithSAMLResult(Instant).

**Expected behavior**
The role given by OktaAwsCliEnvironment is used without prompting the user via System.in

**Additional context**
I think it would be enough to just change one line in:
`AssumeRoleWithSAMLResult getAssumeRoleWithSAMLResult(Instant startInstant)`
from `environment.awsRoleToAssume = currentProfile.map(profile1 -> profile1.roleArn).orElse(null);` to
`environment.awsRoleToAssume = currentProfile.map(profile1 -> profile1.roleArn).orElse(environment.awsRoleToAssume);` (just like in method `run()`)

I'm using a customized version of AWSCredentialsUtil which allows to specify the OktaAwsCliEnvironment programmatically.:
```java
    static AWSSessionCredentials getAWSCredential (OktaAwsCliEnvironment environment) throws Exception
    {
        AssumeRoleWithSAMLResult samlResult = OktaAwsCliAssumeRole.withEnvironment(environment).getAssumeRoleWithSAMLResult(Instant.now());
        Credentials credentials = samlResult.getCredentials();
        return new BasicSessionCredentials(credentials.getAccessKeyId(), credentials.getSecretAccessKey(), credentials.getSessionToken());
    }
```
BTW.: In order to do this in a clean way it would be nice to have that method directly or to make OktaAwsCliAssumeRole#getAssumeRoleWithSAMLResult(Instant) public

**Comment by AlainODea** [comment]
This seems like a good addition to AwsCredentialsUtil. I’m open to this change if you open a pull request with it.

**Comment by froshyfrosh** [comment]
Thanks, I have done that (pull request #247).

**Comment by AlainODea** [comment]
Closed by merging #247
